tests/e2e/taqueria-plugin-flextesa.spec.ts

```diff
@@ -0,0 +1,242 @@
+import {generateTestProject} from "./utils/utils";
+import {execSync} from "child_process";
+import waitForExpect from "wait-for-expect";
+import fs from "fs";
+import {isPortReachable} from "./utils/utils";
+
+
+const taqueriaProjectPath = 'e2e/auto-test-flextesa-plugin';
+let dockerName: string;
+
+describe("E2E Testing for taqueria flextesa plugin",  () => {
+
+    beforeAll(async () => {
+        await generateTestProject(taqueriaProjectPath, ["flextesa"]);
+    })
+
+    test('Verify that taqueria flextesa plugin can start and stop a default sandbox without specifying name', async () => {
+        try {
+
+            // Setting up docker container name
+            dockerName = "local"
+
+            // 1. Run sandbox start command
+            const stdoutStart = execSync(`taq start sandbox`, {cwd: `./${taqueriaProjectPath}`}).toString().trim();
+
+            // 2. Verify that sandbox has been started and taqueria returns proper message into console
+            expect(stdoutStart).toEqual("Started local.");
+
+            // 3. Verify that docker container has been started
+            const [,dockerContainerLocal] = execSync(`docker ps --filter "name=${dockerName}"`).toString().trim().split(/\r?\n/);
+            expect(dockerContainerLocal).toBeDefined();
+
+            // 4.  Run stop command and verify the output
```

Comment:
Can this test have the isPortReachable function applied to it? Since there is not a lot of difference between this test and the next one they should be as similar as possible

Reply:
Sure. Fixed.